playerbot/strategy/actions/AttackAction.cpp

```diff
@@ -116,12 +116,6 @@ bool AttackAction::IsTargetValid(Unit* target)
         if (verbose) ai->TellError(msg.str());
         return false;
     }
-    else if (!sServerFacade.IsWithinLOSInMap(bot, target))
-    {
-        msg << " is not on my sight";
-        if (verbose) ai->TellError(msg.str());
-        return false;
-    }

```

Comment:
Attack action is mostly used for target selection, so LOS checks should not be here